.cirrus.yml

```diff
@@ -638,6 +660,26 @@ upload_snap_task:
         - 'cd contrib/snapcraft && snapcraft && snapcraft push *.snap --release edge'
 
 
+test_docs_task:
+
+    only_if: >-
+        $CIRRUS_BRANCH != $DEST_BRANCH &&
+        $CIRRUS_CHANGE_MESSAGE =~ '.*CI:DOCS.*'
+    depends_on:
+        - "gating"
+        - "vendor"
+          #- "test_docs"
+          #- "varlink_api"
+          #- "build_each_commit"
+    stub_script: '/bin/true'
+
+
+#publish_docs_task:
+#
+#    only_if: $CIRRUS_BRANCH == $DEST_BRANCH &&
+#             $CIRRUS_TAG =~ '^v[0-9]\.[0-9]\.[0-9]$'
```

Comment:
can just nuke this publish section.  easy to add it later when needed.

Reply:
i think i will leave this in case we do need something like this.  then we dont have to ask.